admin_manual/configuration/server/config_sample_php_parameters.rst

```diff
@@ -828,14 +828,31 @@ Options for the Apps folder, Apps store, and App code checker.
 
 The URL of the appstore to use.
 
-Use the ``apps_paths`` parameter to set the location of the Apps directory,
-which should be scanned for available apps, and where user-specific apps
-should be installed from the Apps store. The ``path`` defines the absolute
-file system path to the app folder. The key ``url`` defines the HTTP Web path
-to that folder, starting from the ownCloud webroot. The key ``writable``
-indicates if a Web server can write files to that folder.
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+::
 
+    'apps_paths' => [
```

Comment:
I'm replacing the existing docblock comments in https://github.com/owncloud/core/blob/master/config/config.sample.php#L720 with this.

Reply:
! Pls have the core as default and change this one here.

Imho better to close this PR and let the scrip run successfully to generate a new config_sample_php_parameters.rst from core !